assets/src/infrastructure/ui/display/Alert/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { IAlert } from '@chakra-ui/core';
+
+import Error from './Error';
+
+interface AlertProps extends IAlert {
+	description: string;
+	title: string;
+}
+
+const Alert: React.FC<AlertProps> = ({ description, status, title }) => {
+	if (status === 'error') {
+		return <Error description={description} title={title} />;
+	}
+
+	return null;
+};
```

Comment:
what's the point of this?!?!?

why add a totally new component that either returns another component or nothing ?!?! 

Why would anyone use this?

plz remove this

Reply:
removed